src/shared/api/remoteData.ts

```diff
@@ -1,6 +1,16 @@
 import * as seamlessImmutable from 'seamless-immutable';
 import {MobxPromiseImpl, MobxPromise, MobxPromiseFactory, MobxPromiseInputUnion, hasObservers} from 'mobxpromise';
 
+type errorHandler = (error:Error)=>void;
+
+let errorHandlers: errorHandler[] = [];
+
+export function addErrorHandler(handler: errorHandler) {

```

Comment:
I think it would be cool to put this on the remoteData object, it's SLIGHTLY weird to me that its free floating but adds a global error handler for remoteData.